concrete/controllers/single_page/dashboard/system/attributes/topics.php

```diff
@@ -100,4 +100,36 @@ public function remove_tree_node()
             Loader::helper('ajax')->sendError($this->error);
         }
     }
+
+    public function tree_edit()
+    {
+        $vs = Loader::helper('validation/strings');
```

Comment:
Loader is deprecated: you should use `$vs = $this->app->make('helper/validation/strings');`